assets/js/script.js

```diff
@@ -0,0 +1,990 @@
+const $gameBoard = $(".gameBoard")
```

Comment:
What will be the difference if you wrap the entire code with `$(function() { ... })` ?